.circleci/config.yml

```diff
@@ -268,31 +268,15 @@ workflows:
   version: 2
   test:
     jobs:
-      - pre-checks
-      - linux-python-39:
-          requires:
-            - pre-checks
-      - linux-python-39-minimal:
-          requires:
-            - pre-checks
-      - linux-python-38:
-          requires:
-            - pre-checks
-      - linux-python-37:
-          requires:
-            - pre-checks
-      - linux-python-36-oldest:
-          requires:
-            - pre-checks
-      - linux-pypy-37:
-          requires:
-            - pre-checks
-      - windows-python-38:
-          requires:
-            - pre-checks
-      - macos-python-3:
-          requires:
-            - pre-checks
+#     - pre-checks
```

Comment:
Why do we keep this comment?

Reply:
I was planning to keep this temporarily -- I don't want to remove the `pre-checks` job configuration just yet, until we keep pre-commit CI enabled for a little bit longer.